contracts/Core/AssetManager.sol

```diff
@@ -11,8 +11,8 @@ contract AssetManager is ACL, AssetStorage, Constants {
 
     event JobCreated(
         bool repeat,
-        assetTypes assetType,
```

Comment:
this was removed as there was stack to deep error, due to surpassing a limit 
plus it was given that assetType would be job only as event is jobCreated, so there was no need of it.
```
Compiling 39 files with 0.8.4
CompilerError: Stack too deep, try removing local variables.
  --> contracts/Core/AssetManager.sol:88:103:
   |
88 |         emit JobCreated(repeat, assetTypes.Job, numAssets, power, msg.sender, epoch, block.timestamp, name, selector, url);
   |                                                                                                       ^^^^


Error HH600: Compilation failed

For more info go to https://hardhat.org/HH600 or run Hardhat with --show-stack-traces

```
The same is done for collectionCreated